test/Calendar.js

```diff
@@ -19,18 +19,22 @@ chai.should();
 chai.use(chaiHttp);
 
 describe('Calendar', () => {
-  let calendarStub = null;
+  let getCalendarEventStub = null;
+  let addCalendarEventStub = null;
   before(done => {
-    calendarStub = sandbox.stub(SceGoogleApiHandler.prototype,
+    getCalendarEventStub = sandbox.stub(SceGoogleApiHandler.prototype,
       'getEventsFromCalendar');
+    addCalendarEventStub = sandbox.stub(SceGoogleApiHandler.prototype,
+      'addEventToCalendar');
     app = tools.initializeServer(
       __dirname + '/../api/google_api/routes/Calendar.js');
     test = new SceApiTester(app);
     done();
   });
 
   after(done => {
-    if (calendarStub) calendarStub.restore();
+    if (getCalendarEventStub) getCalendarEventStub.restore();
```

Comment:
you dont need 36 and 37, `sandbox.restore` takes care of it for you, see the [sinon docs](https://sinonjs.org/releases/latest/sandbox/#sandboxrestore)